src/Vendr.PaymentProviders.Worldpay/WorldpaySettings.cs

```diff
@@ -1,10 +1,58 @@
 using Vendr.Core.Web.PaymentProviders;
 
-namespace Vendr.PaymentProviders
+namespace Vendr.PaymentProviders.Worldpay
 {
     public class WorldpaySettings
     {
-        [PaymentProviderSetting(Name = "Continue URL", Description = "The URL to continue to after this provider has done processing. eg: /continue/")]
+        [PaymentProviderSetting(Name = "Cancel Url", Description = "The Cancel URL")]
+        public string CancelUrl { get; set; }
+
+        [PaymentProviderSetting(Name = "Error Url", Description = "The Error URL")]
+        public string ErrorUrl { get; set; }
+
+        [PaymentProviderSetting(Name = "Continue Url", Description = "The Continue URL")]
         public string ContinueUrl { get; set; }
```

Comment:
I would probably re-order these properties so ContinueUrl is first, then Cancel and then Error so that it is consistent with other providers. https://github.com/vendrhub/vendr-payment-provider-stripe/blob/dev/src/Vendr.PaymentProviders.Stripe/StripeSettingsBase.cs#L7-L20